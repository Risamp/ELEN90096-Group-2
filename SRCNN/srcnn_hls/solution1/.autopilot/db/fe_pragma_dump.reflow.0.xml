<pragmas>
<pragma file="src/srcnn.cpp" line="21" pragmaType="pipeline" parentfunction="srcnn" off="1" ii="0" style=""/>
<pragma file="src/conv1.cpp" line="29" pragmaType="unroll" parentfunction="conv1" factor="3" skip_exit_check="0"/>
</pragmas>
