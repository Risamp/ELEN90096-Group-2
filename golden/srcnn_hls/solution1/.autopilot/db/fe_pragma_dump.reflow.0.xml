<pragmas>
<pragma file="src/srcnn.cpp" line="15" pragmaType="pipeline" parentfunction="srcnn" off="1" ii="0" style=""/>
</pragmas>
